pallets/keystore/src/weights.rs

```diff
@@ -0,0 +1,18 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{constants::RocksDbWeight, Weight};
+
+/// Weight functions for pallet_fennel_identity.
+pub trait WeightInfo {
+    //fn create_identity() -> Weight;
+    //fn revoke_identity() -> Weight;
+    //fn add_or_update_identity_trait() -> Weight;
+    //fn remove_identity_trait() -> Weight;
+    //fn sign_for_identity() -> Weight;
+}
+
+impl WeightInfo for () {
```

Comment:
Do we maybe want `impl<T> WeightInfo were T: pallet_keystore::Trait` ? I'm not too familiar with runtime dev but why impl on `()`

Reply:
We had this very fun issue where both were required, or `impl WeightInfo for ()` was required. During testing, the mock setup from the documentation we reviewed failed because WeightInfo wasn't implemented for `()`, which was being used to stand in for a real runtime type.

Reply:
ah i see